src/Services/PrescriptionService.php

```diff
@@ -153,50 +159,6 @@ public function getAll($search = array(), $isAndCondition = true, $puuidBind = n
                     LEFT JOIN
                         -- @brady.miller so drug_id in my databases appears to always be 0 so I'm not sure I can grab anything here.. I know WENO doesn't populate this value...
                         drugs ON prescriptions.drug_id = drugs.drug_id
-                    UNION
```

Comment:
Please explain what problems you ran into and  why you are removing the union on the medications table.  We track certain kinds of prescriptions in this table, it sucks that it is that way, but its how OpenEMR handles it.  This code currently is required for our ONC certification.

Reply:
Here we were getting duplications of the prescription since when you add a prescription through the UI it's added to the medication list of the patient and in the prescriptions table, so we get it twice in the prescriptions response. If you think that this is wrong and it can cause some unexpected behavior, I am happy to move it back. Thanks!